src/EditorFeatures/CSharpTest/Braces/AddBracesTests.cs

```diff
@@ -29,7 +45,7 @@ static void Main()
         [|if|] (true) { return; }
     }
 }
-");
+", options: AddBraces());
```

Comment:
This preferred way to do this pattern in tests is to extract out duplication.  i.e. you could have a TestAddBraces which then calls the core Test method passing along the option.  THen all the tests call TestAddBraces.